crowdsourcehinter/crowdsourcehinter.py

```diff
@@ -157,7 +156,8 @@ def extract_student_answers(self, answers):
         # First, we split this into the submission
         answers = [a.split('=') for a in answers.split("&")]
         # Next, we decode the HTML escapes
-        answers = [(a[0], html_parser.unescape(a[1])) for a in answers]
+        answers = [(a[0], urllib.unquote_plus(a[1])) for a in answers]
+        print answers
```

Comment:
Please remove the print statement, and then let's merge to master. We can continue debugging/resolving issue from there.